src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -962,6 +970,10 @@ protected ParameterEvaluatorService getParameterEvaluatorService(){
     	return SpringContext.getBean(ParameterEvaluatorService.class);
     }
 
+    @Override
+    public void setDvPayeeDetail(DisbursementVoucherPayeeDetail dvPayeeDetail) {
+        this.dvPayeeDetail = (CuDisbursementVoucherPayeeDetail) dvPayeeDetail;
+    }
 
     public void setDvPayeeDetail(CuDisbursementVoucherPayeeDetail dvPayeeDetail) {
```

Comment:
I see that you added an override of the superclass's "setDvPayeeDetail" setter, but the subclass-specific setter is still present. Can the code be updated so that it only has one dvPayeeDetail setter method?

Also, this subclass still contains its own "dvPayeeDetail" variable. Can the subclass's variable be removed so that the code can use the one inherited from the parent class?